test/libsolidity/util/TestFunctionCall.h

```diff
@@ -137,6 +135,8 @@ class TestFunctionCall
 		bool _showDifference
 	) const;
 
+	bool validateFractionDigits() const;
```

Comment:
Can you change the name to be a property instead of an action?

Reply:
Renamed to `validFractionDigits`.